src/garage/experiment/local_runner.py

```diff
@@ -148,6 +148,9 @@ def __init__(self, snapshot_config, max_cpus=1):
         self.step_itr = None
         self.step_path = None
 
+        # only used for off-policy algorithms
+        self.buffer_prefilled = True
```

Comment:
This isn't really a great name, because it describes what the variable is "used to do" but not "what the variable does."
When reading the code, it's much more useful to have variables that describe what they cause to happen.

Perhaps call this `enable_logging` and default it to `True` still.